PostgresAssignment/src/database/index.ts

```diff
@@ -0,0 +1,21 @@
+const { Pool } = require("pg");
+const config = require("../config/config");
+
+const pool = new Pool({
+	user: config.dbUser,
+	password: config.dbPassword,
+	host: config.dbHost,
+	database: config.dbName,
+	max: 20,
+	connectionTimeoutMillsis: 0,
+	idleTimeoutMillsis: 0,
+});
+
+export async function query(querystring: string) {
+	try {
```

Comment:
why try / catch ? here and in services ? we are throwing same error so how this would help ?